src/features/contactmoment/service.ts

```diff
@@ -109,6 +132,94 @@ export function useContactverzoekenByKlantId(
   });
 }
 
+export function useContactmomentenByKlantId(
+  id: Ref<string>,
+  page: Ref<number>
+) {
+  function getUrl() {
+    const searchParams = new URLSearchParams();
+    searchParams.set("klant", id.value);
+    return `${klantcontactmomentenUrl}?${searchParams.toString()}`;
+  }
+
+  const mapObjectContactmoment = (a: any) => {
+    return a as ObjectContactmoment;
+  };
+
+  const objectcontactmomenten = async (
+    x: any,
+    objectcontactmomenten: Array<any>
+  ) => {
+    const zaken = [];
+
+    for (const item of objectcontactmomenten) {
+      const relUrl = toRelativeProxyUrl(item, contactmomentenProxyRoot);
+      if (relUrl) {
+        const objectcontactmoment = await fetchLoggedIn(relUrl)
+          .then(throwIfNotOk)
+          .then((r) => r.json())
+          .then((x) => mapObjectContactmoment(x));
+
+        if (objectcontactmoment.objectType === "zaak") {
+          zaken.push(objectcontactmoment.object);
+        }
+      }
+    }
+
+    return { ...x, zaken };
+  };
+
+  const mapKlantContactmoment = async (
```

Comment:
is de verwachting dat dit async wordt?

Reply:
restant van eerder oplossing. doet eigenlijk niets meer. opgeruimd